rulesets/data/es_lookup_by_cz.csv

```diff
@@ -0,0 +1,276 @@
+mechanical_ventilation_fan_type,,MF_National_1.2,supply only,supply only,supply only,supply only,supply only,supply only,supply only,supply only,supply only,supply only,supply only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only
+mechanical_ventilation_fan_type,,MF_OregonWashington_1.2,,,,,,,,,,,,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,exhaust only,,
+door_ufactor,,SF_National_3.0,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21
+door_ufactor,,SF_National_3.1,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17
+door_ufactor,,SF_National_3.2,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17
+door_ufactor,,SF_Pacific_3.0,0.21,0.21,0.21,,,,,,,,,,,,,,,,,
+door_ufactor,,SF_Florida_3.1,0.21,0.21,0.21,0.21,0.21,0.21,,,,,,,,,,,,,,
+door_ufactor,,SF_OregonWashington_3.2,,,,,,,,,,,,0.17,0.17,0.17,0.17,0.17,0.17,0.17,,
+door_ufactor,,MF_National_1.0,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21,0.21
+door_ufactor,,MF_National_1.1,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17
+door_ufactor,,MF_National_1.2,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17,0.17
+door_ufactor,,MF_OregonWashington_1.2,,,,,,,,,,,,0.17,0.17,0.17,0.17,0.17,0.17,0.17,,
+foundation_walls_ufactor_or_rvalue,,SF_National_3.0,0.36,0.36,0.36,0.36,0.36,0.36,0.091,0.091,0.091,0.059,0.059,0.059,0.059,0.059,0.059,0.05,0.05,0.05,0.05,0.05
+foundation_walls_ufactor_or_rvalue,,SF_National_3.1,0.36,0.36,0.36,0.36,0.36,0.36,0.091,0.091,0.091,0.059,0.059,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05
+foundation_walls_ufactor_or_rvalue,,SF_National_3.2,0.36,0.36,0.36,0.36,0.36,0.36,0.091,0.091,0.091,0.059,0.059,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05
+foundation_walls_ufactor_or_rvalue,,SF_Pacific_3.0,0.36,0.36,0.36,,,,,,,,,,,,,,,,,
+foundation_walls_ufactor_or_rvalue,,SF_Florida_3.1,0.36,0.36,0.36,0.36,0.36,0.36,,,,,,,,,,,,,,
+foundation_walls_ufactor_or_rvalue,,SF_OregonWashington_3.2,,,,,,,,,,,,0.042,0.042,0.042,0.042,0.042,0.042,0.042,,
+foundation_walls_ufactor_or_rvalue,,MF_National_1.0,0,0,0,0,0,0,0,0,0,7.5,7.5,7.5,7.5,7.5,7.5,7.5,7.5,7.5,10,12.5
+foundation_walls_ufactor_or_rvalue,,MF_National_1.1,0,0,0,0,0,0,0,0,0,7.5,7.5,7.5,7.5,7.5,7.5,7.5,7.5,7.5,10,12.5
+foundation_walls_ufactor_or_rvalue,,MF_National_1.2,0.36,0.36,0.36,0.36,0.36,0.36,0.091,0.091,0.091,0.059,0.059,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05
+foundation_walls_ufactor_or_rvalue,,MF_OregonWashington_1.2,,,,,,,,,,,,15,15,15,15,15,15,15,,
```

Comment:
Split this into `foundation_walls_ufactor` and `foundation_walls_rvalue` to make it simpler.